rfd/0017-supply-chain-hardening.md

```diff
@@ -0,0 +1,59 @@
+# Supply Chain Hardening
```

Comment:
This spec could benefit from a few examples: a supply chain compromise attempt and which control stops it.